client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -13,7 +15,7 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 				href={ actionUrl }
 				data-task={ id }
 			>
-				{ task.isCompleted && (
+				{ task.isCompleted && ! isLinkInBioSiteLaunchSiteTask && (
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/67224/files#r959891266

We could read the dependencies array here